src/System.Private.Xml/tests/XmlSerializer/ReflectionOnly/System.Xml.XmlSerializer.ReflectionOnly.Tests.csproj

```diff
@@ -4,12 +4,12 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);ReflectionOnly</DefineConstants>
     <ProjectGuid>{4050F1D1-1DD2-4B48-A17B-E3F955518C4B}</ProjectGuid>
-    <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);FEATURE_SERIALIZATION_UAPAOT</DefineConstants>
```

Comment:
> Good point. Can you please also change the <DefineConstants> in System.Xml.XmlSerializer.Tests.csproj then? Thanks!

Sorry. I was wrong above. This test behavior is uapaot specific. It's not specific for `FEATURE_SERIALIZATION_UAPAOT`. We should change this to `uapaot`.